examples/text_summarization/text_summarization/workflow.py

```diff
@@ -42,13 +42,11 @@ class GroundTruth(BaseGroundTruth):
 @dataclass(frozen=True)
 class Inference(BaseInference):
     summary: str
+    source: str
     word_count: int
     is_failure: bool = False
     inference_time: Optional[float] = None
-    tokens_input_text: Optional[int] = None
```

Comment:
what's the reason for the removal?

Reply:
For demo/example purposes, one token count provides more clarity than 4 different counts. More importantly, the CSVs containing the precomputed metrics do not have the other token counts.